lib/management/configuration.js

```diff
@@ -179,6 +179,8 @@ function patchConfiguration(newConf, log, cb) {
                         l.locationType });
                     return;
                 }
+                location.sizeLimitGB = l.sizeLimitGB || undefined;
```

Comment:
To ease debugging then, let's use `null` which conveys that it was manually assigned but chosen as missing-value, whereas `undefined` usually means that it was never manually defined.